core/client/src/main/java/alluxio/client/file/options/InStreamOptions.java

```diff
@@ -30,6 +32,7 @@
 @PublicApi
 @NotThreadSafe
 public final class InStreamOptions {
+  // TODO(peis): Rename this to mCacheLocationPolicy in 2.0.
```

Comment:
I feel that is an overkill for an getter/setter. I just removed this TODO to avoid confusion.